dev-db/sqlitebrowser/sqlitebrowser-3.9.1.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite."
+DESCRIPTION="It is for users and developers wanting to create databases, search, and edit data. It uses a familiar spreadsheet-like interface, and you don't need to learn complicated SQL commands.\
+Controls and wizards are available for users to:\
+Create and compact database files\
+Create, define, modify and delete tables\
+Create, define and delete indexes\
+Browse, edit, add and delete records\
+Search records\
+Import and export records as text\
+Import and export tables from/to CSV files\
+Import and export databases from/to SQL dump files\
+Issue SQL queries and inspect the results\
+Examine a log of all SQL commands issued by the application"
+HOMEPAGE="https://github.com/sqlitebrowser/sqlitebrowser"
+COPYRIGHT="Copyright notice>"
+LICENSE="GNU GPL v3"
+REVISION="1"
+scrGitRev="70ff691f1a8a82c91c22160c6d990fbee9bc0eb7"
+SOURCE_URI="$HOMEPAGE/archive/$scrGitRev.tar.gz"
+CHECKSUM_SHA256=""
+SOURCE_DIR="sqlitebrowser-$scrGitRev"
+
+ARCHITECTURES="x86_gcc2  x86"
+
+PROVIDES="
+    sqlitebrowser = $portVersion
+    libs:antlr-2.7.7
+    libs:qcustomplot-source
+    libs:qhexedit
+    libs:qscintilla
+    "
+REQUIRES="
+        haiku >= $haikuVersion
+        "
+BUILD_REQUIRES="
+	
+        "
+
+BUILD_PREREQUIRES="
+	haiku_devel
+	cmd:cmake
+	cmd:qt-devel
```

Comment:
cmd:qt-devel will not work. (Sorry for the delay for proviiding feedback about this, btw.)
You need to replace it by "libqt4_devel >= 4.8" or,even better, by some or all of the devel: items listed in the PROVIDES_devel ot dev-qt/libqt4/libqt4-4.8.7.recipe. But you'll need to get rid of the $secondaryArchSuffix , not because it wouldn't work with it (it would), but because we don't need thes $secondaryArchSuffix here.
Then you can build with the -S option of haikuporter. It will tell you which libs are used but missing a declaration in REQUIRES. (For each of these required libs you will need to add a lib:lib line)
Then you should rebuild (you can use "-F -S" to go faster, until haikuporter stops complaining about missing REQUIRES.
After that you can remove from BUILD_REQUIRES each devel:lib line that does not have its corresponding lib:lib line in REQUIRES.
Yep, that's a bit long...

But before you follow the steps above, you need to clean your BUILD_PREREQUIRES from the strange lines (e.g. cmd:sqlite-devel and maybe cmd:ant-antlr too) otherwuse you won't be able to build the recipe, since these strange cmd: lines do not exist.